cmds/core/init/fns_unix.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build unix
```

Comment:
go vet requires this tag, in spite of the name.

go build does not.

Reply:
I don't think Go has a unix tag (yet). It's being discussed upstream I think, but in the meantime you gotta add +build linux here.

Reply:
thanks, I could have sworn this was a thing at some point ...

Reply:
It was just accepted by Russ upstream: https://github.com/golang/go/issues/20322

will be in the next Go sometime

Reply:
so this builds just fine. I think what's confusing is xxx_unix.go at least used to be a thing? I don't recall.